api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -215,6 +221,25 @@ public void exportWorkspace_FocusOnUnderservedPopulation() throws ApiException {
     verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));
   }
 
+  @Test
+  public void exportWorkspace_CreatorInformation() throws ApiException {
+    List<Long> workspaceID = new ArrayList<>();
+    workspaceID.add(1l);
+    rdrExportService.exportWorkspaces(workspaceID);
+    verify(rdrExportDao, times(1)).save(anyList());
```

Comment:
Is there anything else you can verify?

Reply:
If you're using a real DAO, then you should be able to fetch the saved object and compare fields.

Reply:
i actually do not need to verify this, the purpose of this test is to ensure the creator information is different

Reply:
and i am verifying the RdrWorkspace converted object attribute value is matched via
    verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));